src/app/authentication/auth-stack.component.tsx

```diff
@@ -0,0 +1,18 @@
+import { createStackNavigator } from '@react-navigation/stack';
+import React from 'react';
+
+import { LoginScreen } from './login-screen.component';
+import { RegisterScreen } from './register-screen.component';
+import { VerificationScreen } from './sms-verification-screen.component';
+
+const Stack = createStackNavigator();
```

Comment:
You should use TypeScript here to define routes and props (check React Navigation TS guide)